### PR TITLE
Missed code block added in comments

### PR DESCRIPTION
**Pull request by kumkanillam**

### CODE REVIEW

File: packages/ember-routing/lib/system/route.js [hunk 1]
```diff
@@ -2119,7 +2119,7 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
         }
       }
     });
-
+	```
```
Comment (locks) [inline packages/ember-routing/lib/system/route.js]: Can you add a newline after the code block?

Comment (locks) [inline packages/ember-routing/lib/system/route.js]: Can you add a newline after the code block?

**Review by locks — CHANGES_REQUESTED** [review]

File: packages/ember-routing/lib/system/route.js (hunk 1 above)
Comment (kumkanillam) [inline packages/ember-routing/lib/system/route.js]: done @locks

Comment (locks) [inline packages/ember-routing/lib/system/route.js]: Can you squash the commits and prefix the commit with `[DOC release]`?

Comment (kumkanillam) [inline packages/ember-routing/lib/system/route.js]: done @locks

**Review by locks — APPROVED** [review]

**Comment by locks** [comment]
Thank you :)

**Pull request merged**